src/bench_schnorrsig.c

```diff
@@ -26,7 +27,7 @@ typedef struct {
 void bench_schnorrsig_sign(void* arg, int iters) {
     bench_schnorrsig_data *data = (bench_schnorrsig_data *)arg;
     int i;
-    unsigned char msg[32] = "benchmarkexamplemessagetemplate";
+    unsigned char msg[MSGLEN] = "benchmarkexamplemessagetemplate";
```

Comment:
maybe
```suggestion
    unsigned char msg[MSGLEN] = {0};
```
because people could set MSGLEN to < 32

Reply:
done